editor.planx.uk/src/@planx/components/NextSteps/Public.tsx

```diff
@@ -0,0 +1,25 @@
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import { PublicProps } from "@planx/components/ui";
+import React from "react";
+
+import Card from "../shared/Preview/Card";
+import QuestionHeader from "../shared/Preview/QuestionHeader";
+import { NextSteps } from "./model";
+
+type Props = PublicProps<NextSteps>;
+
+export default Component;
+
+function Component(props: Props) {
+  return (
+    <Card handleSubmit={props.handleSubmit} isValid>
+      <QuestionHeader title={props.title} description={props.description} />
```

Comment:
A more flexible option is to use `<QuestionHeader {...props} />` here which will then also automatically pick up help text if it's defined. Otherwise you'll have to keep adding those prop fields manually like `howMeasured={props.howMeasured}` and so on

Reply:
Thanks, updated.